crabs/detector/datamodules.py

```diff
@@ -170,9 +170,10 @@ def _compute_splits(
         """
 
         # Optionally fix the generator for a reproducible split of data
-        generator = None
+        generator_1, generator_2 = None, None
```

Comment:
do we need to create two generator? I am not quite sure myself, but why we can just use the same generator for both split?

Reply:
I decided to use two separate generators in the end after reading online, thinking that is a bit more future proof.